lib/model/actions.dart

```diff
@@ -1,5 +1,8 @@
 import 'dart:async';
 
+import 'package:flutter/foundation.dart';
```

Comment:
notif: Ignore notifications for logged out accounts
    Fixes: #1264

Commit-message nit: add empty line after the summary line

Reply:
Also the commit message should be clear that two things are happening: notifications are ignored for logged-out accounts, and notifications are removed at the time an account is logged out.